fmn/api/handlers/admin.py

```diff
@@ -0,0 +1,51 @@
+import logging
+
+from fastapi import APIRouter, Depends, HTTPException
+from sqlalchemy.ext.asyncio import AsyncSession
+
+from ...database.model import Rule
+from ...messages.rule import RuleUpdateV1
+from .. import api_models
+from ..auth import Identity, get_identity_admin
+from ..database import gen_db_session
+from ..messaging import publish
+
+log = logging.getLogger(__name__)
+
+router = APIRouter(prefix="/admin")
+
+
+@router.get("/rules", response_model=list[api_models.Rule], tags=["users/rules"])
+async def get_disabled_rules(
+    identity: Identity = Depends(get_identity_admin),
+    db_session: AsyncSession = Depends(gen_db_session),
+):
+    db_result = await db_session.execute(Rule.select_related().filter_by(disabled=True))
+    return db_result.scalars().all()
+
+
+@router.patch("/rules/{id}/disabled", response_model=api_models.Rule, tags=["users/rules"])
+async def edit_rule_disabled(
+    id: int,
+    disable: api_models.RuleDisable,
+    identity: Identity = Depends(get_identity_admin),
+    db_session: AsyncSession = Depends(gen_db_session),
+):
+    rule_db = (await db_session.execute(Rule.select_related().filter(Rule.id == id))).scalar()
```

Comment:
The route/method looks odd in a restful API, I’d rather have it be sth. like this:

```
@router.patch("/rules/{id}", response_model=api_models.Rule, tags=["users/rules"])
async def patch_rule(
    id: int,
    rule_patch: RulePatch,  # like Rule, only all optional
    identity ...,
    db_session ...,
):
    ...
    if rule_patch.disabled is not None:
        rule_db.disabled = rule_patch.disabled
    ...
```